Cannot find module _process error on bokehjs build

I'm just starting to learn Javascript and it's tools, so apologies if this is just a usage error or setup issue on my side.

I'm getting this error on `gulp build` for the current git master version:
```
$ gulp build
[15:46:13] Using gulpfile ~/code/bokeh/bokehjs/gulpfile.js
[15:46:13] Starting 'build'...
[15:46:13] Starting 'scripts'...
[15:46:13] Starting 'scripts:generate'...
[15:46:13] Starting 'scripts:eco'...
[15:46:13] Starting 'styles'...
[15:46:13] Starting 'styles:build'...
generate_defaults.py: Wrote src/coffee/common/defaults.coffee with 104 model classes
Wrote src/coffee/widget/defaults.coffee with 21 model classes

[15:46:14] Finished 'scripts:generate' after 865 ms
[15:46:14] Starting 'scripts:coffee'...
[15:46:14] Finished 'scripts:eco' after 1.03 s
[15:46:17] Finished 'styles:build' after 3.6 s
[15:46:17] Starting 'styles:minify'...
[15:46:17] Finished 'styles:minify' after 146 ms
[15:46:17] Finished 'styles' after 3.74 s
[15:46:20] Finished 'scripts:coffee' after 5.5 s
[15:46:20] Starting 'scripts:compile'...
[15:46:20] Finished 'scripts:compile' after 6.32 μs
[15:46:20] Starting 'scripts:build'...
/Users/deil/code/bokeh/bokehjs/node_modules/resolve/lib/sync.js:33
    throw new Error("Cannot find module '" + x + "' from '" + y + "'");
    ^

Error: Cannot find module '_process' from '/Users/deil/code/bokeh/bokehjs/node_modules/coffee-script/lib/coffee-script'
  at Function.module.exports [as sync] (/Users/deil/code/bokeh/bokehjs/node_modules/resolve/lib/sync.js:33:11)
  at DestroyableTransform._transform (/Users/deil/code/bokeh/bokehjs/gulp/tasks/scripts.coffee:99:25)
  at DestroyableTransform.Transform._read (/Users/deil/code/bokeh/bokehjs/node_modules/readable-stream/lib/_stream_transform.js:172:10)
  at DestroyableTransform.Transform._write (/Users/deil/code/bokeh/bokehjs/node_modules/readable-stream/lib/_stream_transform.js:160:12)
  at Labeled.Pipeline._write (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:73:22)
  at doWrite (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:279:12)
  at writeOrBuffer (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:266:5)
  at Labeled.Writable.write (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:211:11)
  at Labeled.ondata (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:572:20)
  at emitOne (events.js:77:13)
  at Labeled.emit (events.js:169:7)
  at readableAddChunk (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:195:16)
  at Labeled.Readable.push (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:162:10)
  at Labeled.Pipeline._read (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:57:31)
  at DestroyableTransform.onreadable (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:64:18)
  at DestroyableTransform.g (events.js:260:16)
  at emitNone (events.js:67:13)
  at DestroyableTransform.emit (events.js:166:7)
  at emitReadable_ (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:456:10)
  at emitReadable (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:450:7)
  at readableAddChunk (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:206:11)
  at DestroyableTransform.Readable.push (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_readable.js:162:10)
  at DestroyableTransform.Transform.push (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:145:32)
  at DestroyableTransform._transform (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/index.js:664:14)
  at DestroyableTransform.Transform._read (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:184:10)
  at DestroyableTransform.Transform._write (/Users/deil/code/bokeh/bokehjs/node_modules/browserify/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:172:12)
  at Labeled.Pipeline._write (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:73:22)
  at doWrite (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:279:12)
  at writeOrBuffer (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:266:5)
  at Labeled.Writable.write (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:211:11)
  at Labeled.ondata (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:572:20)
  at emitOne (events.js:77:13)
  at Labeled.emit (events.js:169:7)
  at readableAddChunk (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:195:16)
  at Labeled.Readable.push (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:162:10)
  at Labeled.Pipeline._read (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:57:31)
  at DestroyableTransform.onreadable (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:64:18)
  at DestroyableTransform.g (events.js:260:16)
  at emitNone (events.js:67:13)
  at DestroyableTransform.emit (events.js:166:7)
  at emitReadable_ (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_readable.js:456:10)
  at emitReadable (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_readable.js:450:7)
  at readableAddChunk (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_readable.js:206:11)
  at DestroyableTransform.Readable.push (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_readable.js:162:10)
  at DestroyableTransform.Transform.push (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_transform.js:145:32)
  at /Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/index.js:76:16
  at Array.forEach (native)
  at sorter (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/index.js:67:14)
  at DestroyableTransform.end [as _flush] (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/index.js:14:9)
  at DestroyableTransform.<anonymous> (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_transform.js:135:12)
  at DestroyableTransform.g (events.js:260:16)
  at emitNone (events.js:67:13)
  at DestroyableTransform.emit (events.js:166:7)
  at prefinish (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_writable.js:450:12)
  at finishMaybe (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_writable.js:458:7)
  at endWritable (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_writable.js:469:3)
  at DestroyableTransform.Writable.end (/Users/deil/code/bokeh/bokehjs/node_modules/deps-sort/node_modules/readable-stream/lib/_stream_writable.js:436:5)
  at Labeled.<anonymous> (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/index.js:46:26)
  at Labeled.g (events.js:260:16)
  at emitNone (events.js:72:20)
  at Labeled.emit (events.js:166:7)
  at finishMaybe (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:460:14)
  at endWritable (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:469:3)
  at Labeled.Writable.end (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_writable.js:436:5)
  at Labeled.onend (/Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:537:10)
  at Labeled.g (events.js:260:16)
  at emitNone (events.js:72:20)
  at Labeled.emit (events.js:166:7)
  at /Users/deil/code/bokeh/bokehjs/node_modules/stream-splicer/node_modules/readable-stream/lib/_stream_readable.js:934:16
  at nextTickCallbackWith0Args (node.js:433:9)
  at process._tickDomainCallback (node.js:403:13)
```

I don't know if it's related at all ... I got two deprecation warnings on `npm install`:
```
$ npm install
npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
npm WARN deprecated npmconf@2.1.1: this package has been reintegrated into npm and is now out of date with respect to npm
```

Help!?

--- a/bokehjs/gulp/labeler.coffee
+++ b/bokehjs/gulp/labeler.coffee
@@ -70,7 +70,9 @@ namedLabeler = (bundle, parentLabels) -> customLabeler bundle, parentLabels, (ro
     .replace(/\.(coffee|js|eco)$/, "")
     .split(path.sep).join("/")
     .replace(/^(src\/(coffee|vendor)|node_modules|build\/js\/tree)\//, "")
-    .replace("browserify/node_modules/process/browser", "_process")
+
+  if modName.indexOf("process/browser") != -1
+    modName = "_process"
 
   if argv.verbose
     gutil.log("Processing #{modName}")

